File: 1/Euler1.c
#include <stdio.h>

int multiple_of(int x, int y) {
  return (x % y) == 0;
}

int main() {
  int answer = 0;
  int i;

  for(i = 1; i < 1000; i++) {
    if(multiple_of(i, 3) || multiple_of(i, 5)) answer += i;
  }
  printf("%i\n", answer);

  return 0;
}

File: 1/Euler1.php
<?php 
function multiple_of($x, $y) {
  return ($x % $y) == 0;
}

$answer = 0;

for($i = 0; $i < 1000; $i++) {
  if(multiple_of($i, 3) || multiple_of($i, 5)) $answer += $i;
}
echo "$answer\n";
?>